translate/test/cmumps-utils-mocha.js

```diff
@@ -18,7 +18,7 @@ describe('for cmumps-utils', function() {
 
     //
     describe('expecting node version', function() {
-        expect(semver.parse(process.versions.node).major).is.greaterThan(4);
+        expect(semver.parse(process.versions.node).major).is.least(4);
```

Comment:
It would be better style to make this more like English: 
expect(semver.parse(process.versions.node).major).to.be.at.least(4);